### PR TITLE
Get redis host from env variable SPDX_RDIS_HOST

### PR DESCRIPTION
**Pull request by kmehant**
Signed-off-by: K mehant <411843@student.nitandhra.ac.in>

### CODE REVIEW

File: src/app/utils.py [hunk 1]
```diff
@@ -30,6 +30,11 @@
 
 from app.models import User, UserID
 
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
```
Comment (rtgdk) [inline src/app/utils.py:36]: Should be load all variable in src/secret.py and then import it here
`from src.secret import getRedisHost()` ?

Comment (kmehant) [reply src/app/utils.py]: @rtgdk Done!

**Comment by goneall** [comment]
> The only thing that is bugging me is SPDX_RDIS_HOST instead of SPDX_REDIS_HOST name. Not sure why @goneall suggested that in #193, maybe a typo. Let @goneall look into this PR and merge.

That was a typo - @kmehant please change to `SPDX_REDIS_HOST` - sorry for the confusion

**Comment by kmehant** [comment]
> > The only thing that is bugging me is SPDX_RDIS_HOST instead of SPDX_REDIS_HOST name. Not sure why @goneall suggested that in #193, maybe a typo. Let @goneall look into this PR and merge.
> 
> That was a typo - @kmehant please change to `SPDX_REDIS_HOST` - sorry for the confusion

Done!

**Comment by goneall** [comment]
Thanks @kmehant !  I'll go ahead and merge.

**Pull request merged**